lib/pr_checker.js

```diff
@@ -169,9 +169,28 @@ class PRChecker {
       logger.warn('No full CI runs detected');
     }
 
+    let lastCiDate = null;
     for (const [type, ci] of ciMap) {
       const name = CI_TYPES.get(type).name;
       logger.info(`Last ${name} CI on ${ci.date}: ${ci.link}`);
+      if (!lastCiDate || lastCiDate < ci.date) {
+        lastCiDate = ci.date;
+      }
+    }
+
+    if (lastCiDate) {
+      let didLogHeader = false;
+      commits.forEach((c) => {
+        const commit = c.commit;
+        if (commit.committedDate > lastCiDate) {
+          if (!didLogHeader) {
+            status = false;
+            didLogHeader = true;
+            logger.warn('Commits pushed after the last CI run:');
```

Comment:
Hmm, on a second thought, we should probably warn about commits after each type of CI, e.g. 2 commits after last CITGM CI, 3 commits after the last full CI, although we need to take into account that some CI cover the others as well (e.g. full CI covers linter CI)..no need to tackle this in this PR though.